hadoop-hdds/framework/pom.xml

```diff
@@ -334,6 +334,20 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>com.coderplus.maven.plugins</groupId>
+        <artifactId>copy-rename-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>rename-generated-config</id>
+            <phase>process-classes</phase>
+          </execution>
+          <execution>
+            <id>rename-generated-test-config</id>
+            <phase>process-test-classes</phase>
```

Comment:
For the two modules, famework  and common, who has this extra "process-test-classes", will both the two generated xml files be loaded during OzoneConfiguration activation? 
If only first generated xml file will be loaded, how TestGeneratedConfigurationOverwrite.java can pass?

Reply:
The config file for test is created in `target/test-classes` (used in their own unit tests) added to `...-tests.jar` (used by some other modules' tests).